acct-user/gopher/gopher-0.ebuild

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="User for gofish"
+ACCT_USER_ID=295
```

Comment:
NACK. Please reserve (fill a PR) a free UID at https://github.com/gentoo/api-gentoo-org/blob/master/files/uid-gid.txt. You can use the script https://github.com/gentoo/api-gentoo-org/blob/master/bin/used_free_uidgids.sh for help.

Reply:
Thanks for the review. Sorry, I did run that script but I forgot to actually reserve what it suggested.